src/app/app-routing.module.ts

```diff
@@ -35,6 +35,11 @@ const routes: Routes = [
   //   path: AppBarRoutes.donate,
   //   loadChildren: './donate/donate.module#DonateModule'
   // },
+  {
+    path: AppBarRoutes.login,
+    loadChildren: () =>
+      import('./core/login/login.module').then(m => m.LoginModule)
+  },
```

Comment:
I don't think it's a good idea to have the login module inside the core module. I would put it as a separate module.